src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -639,5 +619,59 @@ private ContextualFeatureFilterEvaluator GetContextualFeatureFilter(string filte
 
             return filter;
         }
+
+        private async void PublishTelemetry(EvaluationEvent evaluationEvent)
```

Comment:
It looks like this method should take a cancellation token to pass to `PublishEvent`